week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -5,8 +5,10 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
-
+function convertToUSD(price) {
+  price = "$" + (price * 1.4).toFixed(2);
```

Comment:
If I was being **very** picky, then I would say that in general we don't reassign a function parameter to have a different value. Most of the time we make a new variable, just to make it clear that we're changing it. This also means that we can still access the value of the parameter lower in the function if we need to.

Reply:
Thank you for pointing this out. I reassigned according to each currency. :dollar: :bank: